spec/controllers/followings_controller_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'spec_helper'
+
+describe FollowingsController do
+  describe 'POST create' do
+    context 'with authenticated user' do
+      let(:other_user) { Fabricate(:user) }
+      before { set_current_user }
+
+      it 'creates a new following' do
+        post :create, follow_id: other_user
+        expect(Following.first.follow_id).to eq(other_user.id)
+      end
+
+      it 'sets the flash success' do
+        post :create, follow_id: other_user
+        expect(flash[:success]).not_to be_blank
```

Comment:
When possible you should use a positive matcher. In this case it makes sense to do:

```ruby
expect(flash[:success]).to be_present
```